util/tagrepos.py

```diff
@@ -0,0 +1,46 @@
+from github import Github, GithubException
+import sys
+import csv
+
+if len(sys.argv) <> 3:
+    print "Please specify both username and github password."
+    print "Usage: tagrepos.py <username> <password>"
```

Comment:
you need to document usage to include the mention of the fact that you're simply piping the file into python instead of specifying it on the commandline.